targets/psa_switch/psa_switch.cpp

```diff
@@ -66,8 +66,9 @@ extern int import_primitives();
 
 packet_id_t PsaSwitch::packet_id = 0;
 
-PsaSwitch::PsaSwitch(bool enable_swap)
+PsaSwitch::PsaSwitch(port_t max_port, bool enable_swap)
```

Comment:
This PR removes some of the recent changes I did to support arbitrary port numbers (https://github.com/p4lang/behavioral-model/pull/747), it would be great not to undo them

Reply:
Yes, seems like there was some accidental overwrites that we did, I have gone through and made sure they have all been reverted.